conans/unicode.py

```diff
@@ -0,0 +1,20 @@
+import six
+
+
+get_cwd = six.moves.getcwd
+
+
+def make_unicode(v):
+    if six.PY2:
+        if isinstance(v, str):
+            return v.decode("utf8")
```

Comment:
Just curious, how do you know that the str is utf8 and not other encoding like the evil windows one?

Reply:
Good question. I am trying to understand...
```bash
>>> sys.getfilesystemencoding()
'mbcs'
>>> sys.getdefaultencoding()
'ascii'
>>> sys.stdout.encoding
'cp850'
```